common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -432,5 +504,59 @@ color_lists = {
 			125 = "yellow"
 			250 = "red_light"
 		}
+		###### Warcraft Culture Specific CoAs ######
+		### Humans ###
+		special_selection = {
+			trigger = { azerothian_coa_trigger = yes }
+			30  = "white"
+			250 = "yellow"
+		}
+		special_selection = {
+			trigger = { lordaeronian_coa_trigger = yes }
+			100  = "yellow"
+			250  = "white"
+		}	
+		special_selection = {
+			trigger = { arathorian_coa_trigger = yes }
+			50  = "yellow"
+			250 = "white"
+		}
+		special_selection = {
+			trigger = { alteraci_coa_trigger = yes }
+			50  = "yellow_alterac"
+			250 = "orange_light"
+		}
+		special_selection = {
+			trigger = { gilnean_coa_trigger = yes }	
+			50	  = "orange"
+			150   = "yellow"
+			250   = "red_gilneas"
```

Comment:
It should be lighter not to blend with the background red.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/153505736-371dac97-be50-49d6-aff4-151f81b1bc86.png)

</details>

I'm not sure but I think it's `red_gilneas` too.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/153505951-1b1778d5-46d9-411e-a6a8-67c666705be6.png)

</details>

Reply:
Resolved with 895fcae435a93a2fb98e101f42fe89e5a6f2c29e